SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -516,11 +511,6 @@ void CFlowOutput::SetAnalyzeSurface(CSolver **solver, const CGeometry *geometry,
     SetHistoryOutputPerSurfaceValue("AVG_NOX", y_NOx, iMarker_Analyze);
     Tot_Surface_NOx += y_NOx;
     config->SetSurface_NOx(iMarker_Analyze, y_NOx);
-
-    su2double temp = Surface_TotalTemperature_Total[iMarker_Analyze];
-    SetHistoryOutputPerSurfaceValue("AVG_TEMP", temp, iMarker_Analyze);
-    Tot_Surface_Temp += temp;
-    config->SetSurface_Temperature(iMarker_Analyze, temp);
```

Comment:
This commit 69b1e7c2aa046bca6db08c3a0222798e2d4bb170 fixed the last regression test with SURFACE_STATIC_TEMPERATURE. 

I dont see what the purpose here was because  the values from `Surface_TotalTemperature_Total` are taken which are already available as `SURFACE_TOTAL_TEMPERATURE`. The multiplication with reference Temp is missing and `config->SetSurface_Temperature(iMarker_Analyze, temp);` is used for `SURFACE_STATIC_TEMPERATURE`. This last aspect is also where the error comes in I guess. 

I removed this code which added AVG_TEMP now.